Whiskey/Functions/Install-WhiskeyNuGetPackage.ps1

```diff
@@ -0,0 +1,69 @@
+function Install-WhiskeyNuGetPackage
+{
+    <#
+    .SYNOPSIS
+    Downloads and installs NuGet Packages.
+
+    .DESCRIPTION
+    The `Install-WhiskeyNuGetPackage` function uses NuGet.exe to download a NuGet package. Pass the name of the package to the `Name` parameter. By default, the latest non-prerelease version is downloaded. To download a specific version, pass that version to the `Version` parameter. Wildcards are supported. (Please note that if you're using nuget.org as your package source, it has a bug preventing a list of all versions of a package. If you use nuget.org as your package source and your version contains a wildcard, you'll always get the latest version. We recommend you don't use wildcards if you use nuget.org). By default, packages are saved to a `packages` directory in the build root. If you want the packages directory to be in a different directory, pass the path to this different directory to the `DownloadRoot` parameter. The latest version of the NuGet command line is downloaded first. That version of NuGet.exe is then used to install the package.
+    .EXAMPLE
+    Install-WhiskeyNuGetPackage -Name 'NUnit.Runners' -DownloadRoot 'C:\Buildroot\packages'
+
+    Install-WhiskeyNuGetPackage -Name 'NUnit.Runners' -DownloadRoot 'C:\Buildroot\packages' -Version '2.6.4'

```

Comment:
Add an `.EXAMPLE` before the second example, and add a description after each example of what it is demonstrating, e.g. "This example demonstrates...".